amundsen_application/static/js/components/TableDetail/TableIssues/index.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { GlobalState } from 'ducks/rootReducer';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import { Issue } from 'interfaces'; 
+import { getIssues } from 'ducks/issue/reducer'; 
+import { logClick } from 'ducks/utilMethods';
+import { GetIssuesRequest } from 'ducks/issue/types';
+import { truncateText } from '../../../utils/tableissues-utils'; 
+import './styles.scss';
+
+
+export interface StateFromProps {
+  issues: Issue[]; 
+}
+
+export interface DispatchFromProps {
+  getIssues: (key: string) => GetIssuesRequest; 
+}
+
+export interface ComponentProps {
+  tableKey: string;
+}
+
+export type TableIssueProps = StateFromProps & DispatchFromProps & ComponentProps; 
+
+export class TableIssues extends React.Component<TableIssueProps> {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    this.props.getIssues(this.props.tableKey);
+  }
+
+  logIssueClick = (event) => {
+    logClick(event); 
+  }
+
+  renderIssue = (issue: Issue, index: number) => {
+    return (
+      <div className="issue-banner" key={`issue-${index}`}>
+      <a className="issue-link" target="_blank" href={issue.url} onClick={this.logIssueClick}>
```

Comment:
nit: indentation

For `logClick`, we rely try to infer logging metadata. Adding a descriptive `id` such as `table-issue-link-${index}` would help us know what kind of link was clicked. Also you don't need to wrap the `logClick` function unless you have other scripts to run.

Reply:
thanks for the tip! i hadn't noticed the target id was empty